javascript/javascript1/week1/housey-pricey.js

```diff
@@ -0,0 +1,21 @@
+//housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+//let width
+//let depth
+//let height
+//let volumeInMeters = width * depth * height
+//function calculateHousePrice(volumeInMeters, gardenSizeInM2) {
+//  return
+//}
+
```

Comment:
Imagine you can put all values in the same array for both Peter and Julia, for example,
`const volumeInMeters = [8*10*10,5*11*8] `` const gardenSizeInM2 = [100,70]`
where you know that volumeInMeters[0] is Peter's house volume, and another one is Julia's etc.
Can you refactor the rest of your code so you will have fewer lines of code?